### PR TITLE
Mandated git_tag for non-opence remote feedstock

### PR DESCRIPTION
**Pull request by npanpaliya**
## Checklist before submitting

- [ ] Did you read the [contributor guide](https://github.com/open-ce/open-ce/blob/main/CONTRIBUTING.md)?
- [ ] Did you update any affected [documentation](https://github.com/open-ce/open-ce/blob/main/doc/)?
- [ ] Did you write any tests to validate this change?

## Description

Fixes # (issue).

## Review process to land 

1. All tests and other checks must succeed.
2. At least one [maintainer](https://github.com/open-ce/open-ce/blob/main/MAINTAINERS.md) must review and approve.
3. If any  [maintainer](https://github.com/open-ce/open-ce/blob/main/MAINTAINERS.md) requests changes, they must be addressed.

### CODE REVIEW

**Review by alexlang74 — APPROVED** [review]

File: tests/conda_env_file_generator_test.py [hunk 1]
```diff
@@ -135,7 +135,7 @@ def test_variant_specific_env_files():
                    "--cuda_versions", "11.2",
                    "--repository_folder", os.path.join(tmp_test.name, "repos"),
                    "--output_folder", os.path.join(tmp_test.name, "output"),
-                   "https://raw.githubusercontent.com/open-ce/open-ce/open-ce-v1.5.0/envs/opence-env.yaml"]
+                   "https://raw.githubusercontent.com/open-ce/open-ce/open-ce-v1.5.0/envs/tensorflow-env.yaml"]
```
Comment (cdeepali) [inline tests/conda_env_file_generator_test.py:138]: Is this change intentional?

Comment (npanpaliya) [reply tests/conda_env_file_generator_test.py:138]: Yes, this is intentional change. This test case was failing as open-ce-v1.5.0 has one remote feedstock in dali-env.yaml without `git_tag`. So, that can't be fixed now. Hence, I changed it to different env file. The test case is anyway to verify if conda env files are generated for all the combinations of variants.

Comment (cdeepali) [reply tests/conda_env_file_generator_test.py:138]: Okay. Thanks for the clarification.

**Review by cdeepali — APPROVED** [review]
